src/Main.elm

```diff
@@ -116,10 +169,14 @@ view model =
             , div []
                 [ h2 [] [ text "Unwatched Films" ]
                 , div [ class [ Movies ] ]
-                    (List.map Movie.movieCard model.unwatched)
+                    (model.unwatched
+                        |> List.map (Movie.movieCard model.selectedGenres)
```

Comment:
part in parens curries a Movie.movieCard by calling it with just the selected genres. Then List.map passes each movie to curried function to create list with same selectedGenres on each.